### ISSUE TITLE
system: Is systemd-timesyncd.service dead?

### ISSUE THREAD

**Issue by Jeremy7701** [linuxmint/-mint20.2-beta#12]
Running Mint 20.2 beta in a VM, I actually noticed the time drifting from the hosts time.

`systemctl status systemd-timesyncd.service` produced
```
● systemd-timesyncd.service - Network Time Synchronization
     Loaded: loaded (/lib/systemd/system/systemd-timesyncd.service; enabled; vendor preset: enabled)
     Active: inactive (dead)
       Docs: man:systemd-timesyncd.service(8)
```
`systemctl restart systemd-timesyncd.service` followed by `systemctl status systemd-timesyncd.service` produced
```
● systemd-timesyncd.service - Network Time Synchronization
     Loaded: loaded (/lib/systemd/system/systemd-timesyncd.service; enabled; vendor preset: enabled)
     Active: active (running) since Fri 2021-06-18 21:16:16 BST; 37s ago
       Docs: man:systemd-timesyncd.service(8)
   Main PID: 1788 (systemd-timesyn)
     Status: "Initial synchronization to time server 91.189.91.157:123 (ntp.ubuntu.com)."
      Tasks: 2 (limit: 6267)
     Memory: 1.4M
     CGroup: /system.slice/systemd-timesyncd.service
             └─1788 /lib/systemd/systemd-timesyncd

Jun 18 21:16:15 beta systemd[1]: Starting Network Time Synchronization...
Jun 18 21:16:16 beta systemd[1]: Started Network Time Synchronization.
Jun 18 21:16:26 beta systemd-timesyncd[1788]: Initial synchronization to time server 91.189.91.157:123 (ntp.ubuntu.com).

```
***This may well be a VM error** - but it is reproducible in VirtualBox.
Or perhaps a better time-server is supposed to be used - such as `chronyd`
I don't have a spare PC to try this out on.

**Comment by clefebvre** [comment]
> I actually noticed the time drifting from the hosts time

How? If the VM is paused the time will differ indeed. Upon reboot this service should put things back and sync the time again, but only at boot. I don't think it runs continuously.

**Comment by Jeremy7701** [comment]
The VM (in VirtualBox) was continuously active, running in a window so that I can see both the VM and the hosts time. The minute portion normally clicks over simultaneously for host & guest. Also any discrepancy should be gradually adjusted.
The `timedatectl` command displays information about whether the time is synchronised [or not].
I installed the `chrony` package and this cured the VM problem.
```
systemd-timesyncd will be removed
chrony (version 3.5-6ubuntu6.2) will be installed
```
So I conclude that this is is a VM problem?

**Comment by clefebvre** [comment]
I don't know, didn't a VM reboot fix the issue?

**Comment by Jeremy7701** [comment]
At startup time the VM uses the hosts current time.
So if it doesn't run for too long, it's OK.
I installed `chrony` - which removes `systemd-timesyncd` in the Mint 20.2 VM and this fixes the time problem that I observed.

I was not able to boot Mint 20.2 on bare metal - so I can't say if this suffers from this problem. I don't think what happens in the VM is particularly important.

On a non-VM install:-

Run `timedatectl status` - if this says "System clock synchronized: yes" then everything is OK, otherwise not.
The other status information can be ignored.
